source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3768,15 +3769,25 @@ Controls
    :range: 0.0..1.0
 
 
-.. mixxx:control:: [EffectRack1_EffectUnitN],next_chain
-                   [EqualizerRack1_[ChannelI]],next_chain
-                   [QuickEffectRack1_[ChannelI]],next_chain
+.. mixxx:control:: [EffectRack1_EffectUnitN],next_chain_preset
+                   [EqualizerRack1_[ChannelI]],next_chain_preset
+                   [QuickEffectRack1_[ChannelI]],next_chain_preset
 
    Cycle to the next EffectChain preset after the currently loaded preset.
 
    :range: binary
 
 
+.. mixxx:control:: [EffectRack1_EffectUnitN],num_chain_presets
+                   [EqualizerRack1_[ChannelI]],num_chain_presets
+                   [QuickEffectRack1_[ChannelI]],num_chain_presets
+
+   The number of effect chain presets available in this EffectUnit, including the
+   empty/passthrough preset "---".
```

Comment:
inline sourcecode is [two backticks on each side](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup)
```suggestion
   empty/passthrough preset ``---``.
```
Alternatively, [escape from SmartQuotes](https://docutils.sourceforge.io/docs/user/smartquotes.html#toc-entry-2):
```suggestion
   empty/passthrough preset \---.
```

Reply:
Thanks for looking into this. Will try those

Reply:
"\---" does what I want

Reply:
still weird, probably "\-\-\-" is required?
![image](https://github.com/mixxxdj/manual/assets/12380386/54ca0d14-68e0-4272-99a2-3c43aca7e630)

Reply:
:+1:

Reply:
I shouldn't work in bright daylight when I work on visual stuff... :sunglasses: